x-pack/plugins/enterprise_search/common/constants.ts

```diff
@@ -39,6 +39,23 @@ export const ENTERPRISE_SEARCH_CONTENT_PLUGIN = {
   SUPPORT_URL: 'https://discuss.elastic.co/c/enterprise-search/',
 };
 
+export const ANALYTICS_PLUGIN = {
+  ID: 'analytics',
+  NAME: i18n.translate('xpack.enterpriseSearch.analytics.productName', {
```

Comment:
These are OK. They appear to follow the same pattern as other plugins. Just FYI, if you hadn't seen it, here is the guide for namespacing these i18n messages in Kibana: https://github.com/elastic/kibana/blob/main/packages/kbn-i18n/GUIDELINE.md#message-types

Reply:
thanks! bookmarked